docs/devices/FlashingCertificate/Generate.rst

```diff
@@ -24,4 +24,9 @@ Use the IMEI when generating the certificate:
 
     node cli create-device-cert -d "*imei*"
 
+.. note::
+
+    The default lifetime for device certificates is 30 year.
```

Comment:
year -> years

Otherwise looks good to me.

Reply:
Thanks! In the future, could you use the [suggest feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request)? That allows me to incorporate your feedback as intended.

Reply:
```suggestion
    The default lifetime for device certificates is 30 years.
```